### PR TITLE
Fix part #31 : Final UI  MultipleChoiceInteractionView

### LINKED ISSUE #31: Full UI: MultipleChoiceInteractionView [Blocked: #15, #153]
This is tracking implementing the fully polished UI version of the multiple choice interaction.

See the web frontend for details on how the multiple choice interaction works.

Mock: https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/40d6cf5f-f673-42c8-af15-035f711ef6b9/PM-IN-Introduction-

## Checklist
- [ ] Radio Buttons along with text and entire background
Screenshot 2019-11-08 at 3 50 36 PM

- [ ] Final touch up with padding, margin, animations and Accessibility Scanner test

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This Pr corresponds to multiple choice interaction view.

### Mock
https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/191243b4-ba9a-4b4d-a3e3-a894871dfe5b/PM-Q2-Multiple-Choice-No-Answer-

## Accessibility Scanner

- [x]  Accessibility Scanner is working correctly

### Screenshot

Screenshot_20191113-134903

### CODE REVIEW

File: app/src/main/res/drawable/rounded_rect_grey_border_white_background.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?><!-- https://stackoverflow.com/a/16161493/3689782 -->
```
Comment (rt4914) [inline app/src/main/res/drawable/rounded_rect_grey_border_white_background.xml]: Do we need this link everywhere?

File: app/src/main/res/layout/multiple_choice_interaction_items.xml [hunk 2]
```diff
@@ -16,24 +16,27 @@
...
       android:checked="@{viewModel.answerSelected}"
       app:buttonTint="@color/oppiaDarkBlue" />
 
     <TextView
       android:id="@+id/multiple_choice_content_text_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
+      android:layout_marginStart="4dp"
+      android:layout_marginEnd="16dp"
+      android:layout_marginBottom="20dp"
```
Comment (rt4914) [inline app/src/main/res/layout/multiple_choice_interaction_items.xml]: Suggest reducing this margin.

**Review by rt4914 — APPROVED** [review]
LGTM, nit changes, also please change PR name to `Fix part of`

File: app/src/main/res/drawable/rounded_rect_grey_border_white_background.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/drawable/rounded_rect_grey_border_white_background.xml]: Not actually. Removed it.

File: app/src/main/res/layout/multiple_choice_interaction_items.xml (hunk 2 above)
Comment (veena14cs) [reply app/src/main/res/layout/multiple_choice_interaction_items.xml]: I made it 16dp. There is an issue in the mock.  Difference between middle item on top its 29dp and bottom it is 27dp. As gap is more I have adjusted it to 16dp and I have used only bottom margin as 16dp..

**Pull request merged**